defaults/main.yml

```diff
@@ -34,3 +34,9 @@ solr_restart_handler_enabled: true
 # Used only for Solr < 5.
 solr_log_file_path: /var/log/solr.log
 solr_host: "0.0.0.0"
+
+solr_use_systemd: >
```

Comment:
There's a newer variable, I forget exactly what it's called, but it can be used as a simple flag to show whether the OS is using systemd or not... I'd rather do that then set up a hash of all these values here.

Reply:
Ah, I think it's: https://stackoverflow.com/questions/39394333/how-can-i-tell-which-init-system-ansible-runs-when-i-use-the-service-module/39394420#comment87434402_39394420 (`ansible_service_mgr`).